### DISCUSSION TITLE
msvc 下vscode 插件点调试报错，其他功能均正常

### DISCUSSION THREAD

**Discussion by Keeyuu** [xmake-io/xmake#2454] [General]
Error processing 'configurationDone' request. Unable to start the Microsoft Visual Studio Debug Console.
Windows 找不到文件C:/Program ，请确定文件名是否正常后，再试一次

vs版本是2022

**Comment by waruqi** [comment]
你把项目从 c: program files 挪到其他地方，再试下，估计是路径空格导致

**Reply by Keeyuu** [reply]
大佬nb，秒回，我把文件夹移到根目录了，但是还是显示找不到C:/Program

**Reply by Keeyuu** [reply]
但是编译运行都没有问题，完全正常

**Reply by waruqi** [reply]
或者，你拉下插件源码，加载后直接调下：https://github.com/xmake-io/xmake-vscode

**Reply by Keeyuu** [reply]
else if (os.platform() == "win32") {
                    debugConfig = {
                        name: `launch: ${targetName}`,
                        type: 'cppvsdbg',
                        request: 'launch',
                        program: targetProgram,
                        args: args,
                        stopAtEntry: true,
                        cwd: targetRunDir,
                        environment: [],
                        // externalConsole: true, // https://github.com/microsoft/vscode-cpptools/issues/6939
                        //console: "externalTerminal",
                        //MIMode: "gdb",
                        //miDebuggerPath: "",
                        //description: "Enable pretty-printing for gdb",
                        //text: "-enable-pretty-printing",
                        //ignoreFailures: true
                    };
                }
            }

**Reply by Keeyuu** [reply]
我拉了源码，把debugger.ts，这里面注释了就好使，不知道什么原因

**Reply by Keeyuu** [reply]
我怀疑可能是console那里

**Reply by Keeyuu** [reply]
只注释console那里就可以使用了

**Comment by Keeyuu** [comment]
PS C:\Cc> zsh
➜  Cc pwd
/c/Cc
➜  Cc xmake create -P hello
create hello ...
  [+]: src\main.cpp
  [+]: xmake.lua
  [+]: .gitignore
create ok!

**Comment by Keeyuu** [comment]
> Executing task: xmake f -p windows -a x86 -m debug -o "c:\Cc\hello/build"  <

checking for Microsoft Visual Studio (x86) version ... 2022

终端将被任务重用，按任意键关闭。

> Executing task: xmake -w <

[ 25%]: ccache compiling.debug src\main.cpp
[ 50%]: linking.debug hello.exe
[100%]: build ok!

终端将被任务重用，按任意键关闭。

**Reply by waruqi** [reply]
然后呢，没看出什么问题

**Reply by Keeyuu** [reply]
这些流程都是正常的，但是一点调试，就出现Windows 找不到文件C:/Program

**Reply by Keeyuu** [reply]
https://imgtu.com/i/XfSIP0      
这里有张截图
